src/setup.py

```diff
@@ -1,11 +1,22 @@
 import setuptools
```

Comment:
Make sure to add all the necessary dependencies such as `pandas`, `tensorflow`, `keras`, etc.

Reply:
Why do those need to be here? I don't see those in the `setup.py` for other resources. The workflow is already functional without doing this.

Reply:
@simonwa7 Any updates here?

Reply:
Most other resources do have the dependencies in their `setup.py`. The current "best-practice" is that any package explicitly imported within the resource should be included. 

This is particularly because these packages will be installed at runtime in the container running this task if they don't exist in the docker-image (which might happen if someone used a different docker image than `z-lstm`), but is also very useful for testing locally  without having to manually install all of the packages needed.

Reply:
See `install_requires` lines in z-quantum-core's setup.py for example

Reply:
Jumping in this discussion 🪂as I worked with an import bug Javier was having. I have added imports to `install_requires`, a small contribution to a great job by AJ.